dnf/match_counter.py

```diff
@@ -80,9 +74,13 @@ def _key_func(self):
         """
         max_length = self._max_needles()
```

Comment:
As lgtm says, this variable is unused so you can delete it.

Reply:
will do it